.env.dist

```diff
@@ -1,3 +1,6 @@
+COMPOSER_DEV_ARG=
```

Comment:
this is named `.env.dist` so it won't be read by `docker-compose` (or is it?)

Is it meant to be copied onto `.env` for customization? If so `.env` should be ignored?

Reply:
It is ignored https://github.com/libero/browser/blob/master/.gitignore#L2

Reply:
Although this file is overloaded: it contains partly variables for `docker-compose.yml` files and partly variables for the containers in it to use

Reply:
Thinking about it, running `composer install` _outside_ saw `.env` created for me (based on `.env.dist`). Must be some Symfony Flex step. Taking a look.

Reply:
We can set defaults in `docker-compose.yaml` (means repeating it though).

Reply:
We can override the file's values if need be with `source some-file.env && docker-compose up`, but no problem in using a single file if there are no variables named the same way and they are separated in two sections like in this example.

Reply:
Will add a comment to say these are Docker-Composer envvars (if you're using the container elsewhere you won't need these ones).